app/uk/gov/hmrc/customs/emailfrontend/controllers/ChangingYourEmailController.scala

```diff
@@ -78,8 +83,30 @@ class ChangingYourEmailController @Inject() (
           Future.successful(Redirect(routes.CheckYourEmailController.problemWithService()))
       }
 
+  private def sendEmailToCurrentAndNewEmailAddress(details: EmailDetails)(implicit
+    request: Request[AnyContent]
+  ) =
+    List(details.currentEmail.getOrElse(emptyString), details.newEmail).map { emailAddress =>
+      logger.info(s"Email is being sent to $emailAddress")
```

Comment:
Is the logger suppose to have been removed?

Reply:
No, it has been added on purpose to track the sending of email and error (if it occurs)